app/addons/components/react-components.react.jsx

```diff
@@ -1203,6 +1205,183 @@ function (app, FauxtonAPI, React, ReactDOM, Stores, FauxtonComponents, Helpers,
     }
   });
 
+
+  function connectToStores (Component, stores, getStateFromStores) {
+
+    var WrappingElement = React.createClass({
+
+      componentDidMount: function () {
+        stores.forEach(function (store) {
+          store.on('change', this.onChange, this);
+        }.bind(this));
+      },
+
+      componentWillUnmount: function () {
+        stores.forEach(function (store) {
+          store.off('change', this.onChange);
+        }.bind(this));
+      },
+
+      getInitialState: function () {
+        return getStateFromStores(this.props);
+      },
+
+      onChange: function () {
```

Comment:
connectToStores is nice and clean, very cool. 

Do we need an `this.isMounted()` check here?